translation/de_DE.po

```diff
@@ -0,0 +1,2110 @@
+msgstr "Hilbert Kurve"
+
+#: xs/src/libslic3r/PrintConfig.cpp:21
+msgid "Archimedean Chords"
+msgstr "Archimedische Sehnen"
+
+#: xs/src/libslic3r/PrintConfig.cpp:22
+msgid "Octagram Spiral"
+msgstr "Octagramspirale\t"
+
+#: xs/src/libslic3r/PrintConfig.cpp:27
+msgid "Avoid crossing perimeters"
+msgstr "Vermeide Umfangüberquerungen"
+
+#: xs/src/libslic3r/PrintConfig.cpp:28 xs/src/libslic3r/PrintConfig.cpp:77
+#: xs/src/libslic3r/PrintConfig.cpp:252 xs/src/libslic3r/PrintConfig.cpp:259
+#: xs/src/libslic3r/PrintConfig.cpp:544 xs/src/libslic3r/PrintConfig.cpp:712
+#: xs/src/libslic3r/PrintConfig.cpp:726 xs/src/libslic3r/PrintConfig.cpp:820
+#: xs/src/libslic3r/PrintConfig.cpp:842 xs/src/libslic3r/PrintConfig.cpp:900
+#: xs/src/libslic3r/PrintConfig.cpp:1080 xs/src/libslic3r/PrintConfig.cpp:1230
+#: xs/src/libslic3r/PrintConfig.cpp:1442 xs/src/libslic3r/PrintConfig.cpp:1503
+msgid "Layers and Perimeters"
+msgstr "Schichten und Umfänge"
+
+#: xs/src/libslic3r/PrintConfig.cpp:29
+msgid ""
+"Optimize travel moves in order to minimize the crossing of perimeters. This "
+"is mostly useful with Bowden extruders which suffer from oozing. This "
+"feature slows down both the print and the G-code generation."
+msgstr ""
+"Optimire bewegungen um Umfangüberquerungen zu vermeiden.  Das ist "
+"hauptsächlich nützlich mit sickernden Bowden Extrudern.  Dass verlangsamt "
+"das Drucken und auch die G-Codegenerierung."
+
+#: xs/src/libslic3r/PrintConfig.cpp:34
+msgid "Bed shape"
+msgstr "Druckbrettprofil"
+
+#: xs/src/libslic3r/PrintConfig.cpp:44
+msgid "Has heated bed"
+msgstr "Hat Druckbrettheizung"
+
+#: xs/src/libslic3r/PrintConfig.cpp:45
+msgid ""
+"Unselecting this will suppress automatic generation of bed heating gcode."
+msgstr "Deaktivirung schaltet Generierung von druckbrett heizungs G-Code aus."
+
+#: xs/src/libslic3r/PrintConfig.cpp:50 xs/src/libslic3r/PrintConfig.cpp:1428
+msgid "Other layers"
+msgstr "Andere Schichten"
+
+#: xs/src/libslic3r/PrintConfig.cpp:51
+msgid "Bed temperature for layers after the first one."
+msgstr "Druckbrettemperatur für Schichten nach der Ersten."
+
+#: xs/src/libslic3r/PrintConfig.cpp:53
+msgid "Bed temperature"
+msgstr "Druckbrettemperatur"
+
+#: xs/src/libslic3r/PrintConfig.cpp:59
+msgid "Before layer change G-code"
+msgstr "Vorschichtwechsel G-Code"
+
+#: xs/src/libslic3r/PrintConfig.cpp:60
+msgid ""
+"This custom code is inserted at every layer change, right before the Z move. "
+"Note that you can use placeholder variables for all Slic3r settings as well "
+"as [layer_num] and [layer_z]."
+msgstr ""
+"Dieser G-Code wird bei jedem Schichtwechsel gerade vor der Z Bewegung "
+"eingefügt.  Man kann Platzhalter für alle Slic3r Einstellugnen, sowie auch "
+"[layer_num] und [layer_z] benutzen."
+
+#: xs/src/libslic3r/PrintConfig.cpp:68 xs/src/libslic3r/PrintConfig.cpp:76
+msgid "Bottom"
+msgstr "Boden"
+
+#: xs/src/libslic3r/PrintConfig.cpp:69
+msgid "Bottom infill pattern"
+msgstr "Boden Infill Muster"
```

Comment:
Yeah.  "Füllung", "Infill" and "Ausfachung" are plausible translations for "infill".  Translation is hard.